client/jetpack-cloud/sections/partner-portal/assign-license-form/index.tsx

```diff
@@ -64,6 +83,13 @@ export default function AssignLicenseForm( { sites }: any ): ReactElement {
 	const onAssignLater = () =>
 		page.redirect( addQueryArgs( { highlight: licenseKey }, '/partner-portal/licenses' ) );
 
+	const onPageClick = useCallback(
+		( pageNumber: number ) => {
+			setPage( pageNumber );
+		},
+		[ setPage ]
+	);
+
```

Comment:
This seems unnecessary - we can pass `setPage` as the event handler directly instead of proxying it through a callback that doesn't add anything.

Reply:
Now calling `setPage` from within the event handler itself and removed the `onPageClick` callback declaration.